apps/datahub/src/app/record/header-record/header-record.component.html

```diff
@@ -27,6 +27,8 @@
           class="language-switcher text-[13px] mt-0.5"
           [style.--color-main]="foregroundColor"
           [style.--color-gray-300]="foregroundColor"
+          [style.--color-primary-darker]="foregroundColor"
```

Comment:
Is `foregroundColor` defined here?

Reply:
it's defined here: https://github.com/geonetwork/geonetwork-ui/blob/1abd2ec22c087fef3196566f9413395ca8105b88/apps/datahub/src/app/record/header-record/header-record.component.ts#L20